src/MALA.jl

```diff
@@ -11,17 +11,20 @@ MALA(d::RandomWalkProposal) = MALA{typeof(d)}(d)
 MALA(d) = MALA(RandomWalkProposal(d))
 
 
-struct GradientTransition{T<:Union{Vector, Real, NamedTuple}, L<:Real, G<:Union{Vector, Real, NamedTuple}} <: AbstractTransition
+struct GradientTransition{T<:Union{Vector, Real, NamedTuple}, L<:Real, G<:Union{Vector, Real, NamedTuple},M<:Bool,D<:Int} <: AbstractTransition
     params::T
     lp::L
     gradient::G
+    accepted :: M
+    accepted_draws :: D
+    total_draws :: D
```

Comment:
```suggestion
    accepted :: Bool
    accepted_draws :: Int
    total_draws :: Int
```

As suggested above, maybe only store `accepted`?